pyrato/parametric.py

```diff
@@ -5,6 +5,37 @@
 """
 import numpy as np
 
+def calculate_speed_of_sound(temperature):
+    r"""Calculate the speed of sound in air depending on the temperature.
+
+    Parameters
+    ----------
+    temperature : double
+        Temperature in degrees Celsius.
+
+    Returns
+    -------
+    speed_of_sound : double
+        Speed of sound in m/s.
+
+    References
+    ----------
+    .. [#] ISO 9613-1 (Formula A.5)
+    
+    Equations
+    ---------
+    .. math::
+        c = c_0 \sqrt{\frac{temperature - T_0}{20 - T_0}}
+    .. math::
+        T_0=-273.15
+    .. math::
+        c_0 = 343.2
```

Comment:
It might make sence to add a unit here or explain the exact meaning in an additional text